perllib/FixMyStreet/Cobrand/Bristol.pm

```diff
@@ -122,9 +125,37 @@ sub categories_restriction {
     ] } );
 }
 
+
+sub dashboard_export_problems_add_columns {
+    my ($self, $csv) = @_;
+
+    $csv->add_csv_columns(
+        (
+            staff_role => 'Staff Role',
+        )
+    );
+
+    my $user_lookup = $self->csv_staff_users;
```

Comment:
You can return if $csv->dbi is set here (see e.g. Bromley) because staff_role is automatically included in the pre-gen lookup by the flag you've changed in the other file.